lambdas/service/.chalice/config.json.template.py

```diff
@@ -19,8 +21,10 @@
         **{k: v for k, v in os.environ.items() if k.startswith('AZUL_') and k != 'AZUL_HOME'},
         # Hard-wire the ES endpoint, so we don't need to look it up at run-time, for every request/invocation
         'AZUL_ES_ENDPOINT': f'{host}:{port}',
+        'GIT_COMMIT_SHA': f"{repo.head.object.hexsha}",
+        'GIT_REPO_DIRTY': f"{repo.is_dirty()}",
```

Comment:
Suggestion: 

`f"{repo.is_dirty()}"`
`str(repo.is_dirty())`

are equally long but I find the latter more readable.